Resources/keybinds.yml

```diff
@@ -171,6 +171,26 @@ binds:
   key: X
   mod1: Shift
   # DeltaV - Swap Hands Reversed End
+  # DeltaV - NanoChat keybinds Start
```

Comment:
remove middle comments and change outer to Begin DeltaV Additions / End DeltaV Additions
no reason to have separate comments for 1 thing